webpack.common.js

```diff
@@ -36,5 +36,8 @@ module.exports = () => {
         SENTRY_DSN: JSON.stringify(process.env.SENTRY_DSN),
       }),
     ],
+    experiments: {
+      asyncWebAssembly: true,
```

Comment:
Have you tried on staging if that worked?

Reply:
Yes, the version deployed to staging worked.

Reply:
niiice